test/client.nim

```diff
@@ -0,0 +1,65 @@
+import # nim libs
+  json, options, os, times, unittest
+
+import # vendor libs
+  chronos, confutils, json_serialization, sqlcipher, web3/conversions as web3_conversions
+
+import # nim-status libs
+  ../nim_status/[accounts, client, config, conversions, database, settings],
+  ./test_helpers
+
+procSuite "client":
+  asyncTest "client":
+
+    let config = StatusConfig.load()
+    let statusObj = init(config)
+
+    var account:Account = Account(
+      name: "Test",
+      loginTimestamp: 1,
+      identicon: "data:image/png;base64,something",
+      keycardPairing: "",
+      keyUid: "0x1234"
+    )
+
+    saveAccount(statusObj.accountsDB, account)
```

Comment:
In client we could have a:
```nim
proc saveAccount*(self: statusObj, account: Account) =
  saveAccount(self.accountsDB, account)

# usage
statusObj.saveAccount(account)
```

And similar could be done for other procs where `statusObj.accountsDB` is being passed as the first argument. It might not make sense for every such proc, but the basic idea is for the client.nim module to export an API that makes working with a client instance convenient/natural.